src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import url from "url";
+import {Container, Nav, Navbar} from "react-bootstrap";
+import {Link, NavLink, useLocation} from "react-router-dom";
+import "./styles/NavigationBar.scss";
+
+const logo = url.resolve(process.env.PUBLIC_URL, "/icons/se-logo-trimmed.png");
+
+const NavigationBar = (props) => {
+  let location = useLocation();
+  let color = "";
+  switch (location.pathname) {
+  case "/":
+    color = "#F0EDF1";
+    break;
+  default:
+    color = "transparent";
+    break;
+  }
+
+  return (
+    <Navbar expand="sm" collapseOnSelect style={{backgroundColor:color}}>
```

Comment:
Change this line to: `<Navbar expand="sm" collapseOnSelect style={{backgroundColor: color}}>`